setup.py

```diff
@@ -1,12 +1,7 @@
 #!/usr/bin/env python
 
-import sys
-
 from setuptools import setup, find_packages
 
-install_requires = []
-if sys.version_info < (2, 7):
-    install_requires.append('argparse')
```

Comment:
Now that we've got rid of this, we can add a `setup.cfg` that specifies universal wheels.